src/utils/useLayoutChecks.js

```diff
@@ -0,0 +1,50 @@
+import useMediaQuery from './useMediaQuery';
+import { createContext, useContext } from 'react';
+
+const context = createContext({});
+
+const queries = {
+  isDesktop: '(min-width: 600px)', // >= 600px
+  isLaptop1200: '(min-width: 1200px)', // >= than 1200px. -- FIXME(eth-p): Why is this larger than the desktop one?
+  isLaptop1392: '(min-width: 1392px)', // >= than 1392px. -- FIXME(eth-p): Why do we have two?
```

Comment:
Good questions in the comments, unfortunately I'm in no better position to answer these questions 😅
The `1200` and `1392` are a little bit arbitrary to me though. What is the drive behind the choice of these numbers? Do they refer to a group of devices with the same screen size...? 🤔

Reply:
I honestly have no idea. I took the values straight from the code I refactored, but I figured there's some layout-related reason for having two values (e.g. preventing unnecessary wrapping). @joshpeng1999 would be in a better position to answer that, though.